docs/DeveloperGuide.md

```diff
@@ -16,28 +16,41 @@ Take a look at our design which is mostly based off on [AddressBook Level 3 (AB3
 
-Given below is an example of how the `favourite`mechanism behaves with the `favourite`window.
-
-Step 1. The user starts the application with pre-loaded data of Persons.
-
-Step 2. Assuming there is a Person with the number 1. User then executes `favourite 1` command to favourite the first Person in the application. The system will create a new Person with the ‘favourite’ attribute set as true. Then calls `Model#setPerson()` to set Person 1 to be a favourited Person.
+Do note that `EditCommand` will not be able to edit the `Favourite` status of a Person.
 
 <aside>
-💡 **Note:** Every newly added Person will have the default value of `False` for ‘Favourite’ attribute, thus will never appear in the FavouriteWindow before the `favourite` command is called on them.
+💡 **Note:** Previously, to access the Favourites Window, users will navigate to the 'Favourite' menu item under 'File' menu as shown below. Since we wanted to make this app more CLI-friendly, we created a new command 'fw' to open the Favourites Window instead.
 
+![FavouriteWindow](images/developer-guide/FavouriteWindowAccess.png)
 </aside>
 
-Step 3. User can access the `Favourite`Window by navigating to the menu item as shown in the diagram, which pops up a new window that contains only those Persons that have ‘Favourite’ attribute set as True.
+We decided on `fw` as the command for opening up the Favourites Window because this is much shorter and faster to type compared to `favouriteswindow` for example.
 
-![FavouriteWindow](images/developer-guide/FavouriteWindowAccess.png)
+## Property
+The `Property` is a new attribute that can be added to a `Person` that represents a real estate property listing. A `Person` is able to hold multiple properties including none.
+
+The `Property` itself consists of the following attributes: `Region`, `Address`, `Size`, `Price`.
+
+- `Region` represents the general location of the `Address` of the `Property`. It is an enum that can be one of the following: `NORTH`, `SOUTH`, `EAST`, `WEST`, `CENTRAL`.
+
+- `Address` represents the exact location of the `Property`.
+
+- `Size` represents the size of the `Property` in terms of the number of rooms it has. It is an enum that can be one of the following: `ONE_ROOM`, `TWO_ROOM`, `THREE_ROOM`, `FOUR_ROOM`, `FIVE_ROOM`.
+
+- `Price` represents the price of the `Property`. It's `toString()` method returns the price in the form of `$###` or `$###.#K` or `$###.#M` depending on the value of the `Price` for easier readability.
+
+## UserType
+The `UserType` represents an attribute that needs to be added to a `Person` & represents the `Person` as a `buyer` or `seller`. A `Person` is only either a `buyer` or `seller` at a given time. They cannot be both or none.
+
+Similar to other attributes of a `Person`, the `UserType` of a `Person` can be edited via the `edit` command. A `Person` can be changed from a `buyer` to a `seller` & vice versa. This can be done with the command: `edit INDEX t/seller`
```

Comment:
Duplicate section?

Reply:
I don't understand. Which part is duplicating?

Reply:
There is another Property section below in the document with the same content

Reply:
oh ok thanks, I think I did not remove it during merging of conflicts, I will do so now.